mne/channels/tests/test_montage.py

```diff
@@ -43,10 +43,14 @@
                     read_fiducials, __file__ as _MNE_IO_FILE)
 
 from mne.io import RawArray
-from mne.datasets import testing
+from mne.datasets import testing, sample
```

Comment:
This still uses `sample` this should not be necessary...

Reply:
Hmm do you think I should keep it for the sample MEG data? I'm using `fif_fname = op.join(io_dir, 'tests', 'data', 'test_raw.fif')` right now mainly otherwise. I'm not familiar with what's in the testing data, so if that is sufficient, then sounds good!

Reply:
You can use sample_audvis_trunc_raw.fif in the testing dataset, it has the same channels it has just been truncated